app/controllers/buses_controller.rb

```diff
@@ -0,0 +1,49 @@
+class BusesController < ApplicationController
+  before_action :find_bus_by_id, only: %i[edit update destroy]
+
+  def index
+    @buses = Bus.all
```

Comment:
you can show them in ascending order.
```suggestion
    @buses = Bus.all.order("id")
```